javascript-data-model/order-history.js

```diff
@@ -0,0 +1,69 @@
+var orderHistory = [
+  {
+    date: 'August 4, 2020',
+    total: 34.00,
+    recipient: 'JS Masher',
+    'order #': 114 - 3941689 - 8772232,
```

Comment:
Oh dear, never have property names with spaces in them, let alone special characters, if it can be avoided. I would rename this to `orderNumber`

Reply:
Also, this number should be a string, JavaScript will interpret this as math, the value will be one number that is the result of this equation.